docs/web/query/getting-started.mdx

```diff
@@ -21,46 +21,55 @@ export const Example: FC = () => {
+The [code generator](https://www.npmjs.com/package/@connectrpc/protoc-gen-connect-query) does all the work of turning your Protobuf file into something you can easily import. TypeScript types all populate out-of-the-box. Your documentation is also converted to [TSDoc](https://tsdoc.org/).
 
-One of the best features of this library is that once you write your schema in Protobuf form, the TypeScript types are generated and then inferred.  You never again need to specify the types of your data since the library does it automatically.
+One of the best features of this library is that once you write your schema in Protobuf form, the TypeScript types are generated and then inferred. You never again need to specify the types of your data since the library does it automatically.
 
-<div style={{ display: 'flex', justifyContent: 'center', marginTop: 50 }}>
-  <video width="700px" controls><source src="https://user-images.githubusercontent.com/15232461/212362170-65eb9efa-faa5-46fd-ac61-080719d011a5.mp4" /></video>
+<div style={{ display: "flex", justifyContent: "center", marginTop: 50 }}>
+  <video width="700px" controls>
+    <source src="https://user-images.githubusercontent.com/15232461/212362170-65eb9efa-faa5-46fd-ac61-080719d011a5.mp4" />
+  </video>
 </div>
 
 <sup>
   <p align="center">
-    <em>
-      play the above video to see the TypeScript types in action
-    </em>
+    <em>play the above video to see the TypeScript types in action</em>
   </p>
 </sup>
 <dl></dl>
 
-
 ## Generated Code
 
-This example shows the best developer experience using code generation.  Here's what that generated code looks like:
+This example shows the best developer experience using code generation. Here's what that generated code looks like:
 
 ```ts title="your-generated-code/example-ExampleService_connectquery"
-import { createQueryService } from "@connectrpc/connect-query";
+import {
+  createQueryService,
+  createUnaryHooks,
+} from "@connectrpc/connect-query";
 import { MethodKind } from "@bufbuild/protobuf";
 import { ExampleRequest, ExampleResponse } from "./example_pb.js";
 
-export const example = createQueryService({
-  service: {
-    methods: {
-      example: {
-        name: "Example",
-        kind: MethodKind.Unary,
-        I: ExampleRequest,
-        O: ExampleResponse,
-      },
+export const typeName = "your.company.com.example.v1.ExampleService";
+
+export const ExampleService = {
+  methods: {
+    example: {
+      name: "Example",
+      kind: MethodKind.Unary,
+      I: ExampleRequest,
+      O: ExampleResponse,
     },
-    typeName: "your.company.com.example.v1.ExampleService",
   },
-}).example;
+  typeName,
+} as const;
+
+const $queryService = createQueryService({ service: ExampleService });
+
+export const say = {
+  ...$queryService.say,
+  ...createUnaryHooks($queryService.say),
```

Comment:
Not sure if you missed changing this one. Should this be `example`?

Reply:
Ah yeah i def missed that one. Thought I got them all.